Content.Server/Chat/Managers/ChatSanitizationManager.cs

```diff
@@ -9,6 +9,7 @@ public sealed class ChatSanitizationManager : IChatSanitizationManager
 {
     [Dependency] private readonly IConfigurationManager _configurationManager = default!;
 
+    // TODO: this should be defined in YAML instead
```

Comment:
This should definately be moved to yaml prototypes in a future PR by someone, especially since it would allow for multiple different filter lists to be used at the same time.